hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationStatus.java

```diff
@@ -120,6 +124,28 @@ public void testReplicationStatus() throws Exception {
     assertEquals(PEER_ID2, rLoadSourceList.get(0).getPeerID());
   }
 
+  @Test
+  public void testReplicationStatusSink() throws Exception {
+    try (Admin hbaseAdmin = UTIL2.getConnection().getAdmin()) {
+      ServerName server = UTIL2.getHBaseCluster().getRegionServer(0).getServerName();
+      ReplicationLoadSink loadSink = getLatestSinkMetric(hbaseAdmin, server);
+      //First checks if status of timestamp of last applied op is same as RS start, since no edits
+      //were replicated yet
+      assertEquals(loadSink.getTimestampStarted(), loadSink.getTimestampsOfLastAppliedOp());
+      //now insert some rows on source, so that it gets delivered to target
+      insertRowsOnSource();
+      Thread.sleep(10000);
```

Comment:
Although 10s seems more than enough time, we can use `Waiter.wait()` / `HBASE_TESTING_UTILITY.waitFor()` with predicate to check if `loadSink.getTimestampsOfLastAppliedOp()>loadSink.getTimestampStarted()`.